src/wlapplication_messages.cc

```diff
@@ -19,178 +19,207 @@
+	print_help_option(_("--editor"), _("Directly starts the Widelands editor."));
+	print_help_option(_("--editor=FILENAME"),
+	                  _("Directly starts the Widelands editor and loads the map FILENAME."));
+	print_help_option(
+	   _("--ai_training"),
+	   _("Enables AI training mode. See https://www.widelands.org/wiki/Ai%20Training/ for a full "
+	     "description of the AI training logic."));
+	print_help_option(
+	   _("--auto_speed"),
+	   _("In multiplayer games only, this will keep adjusting the game speed automatically, "
+	     "depending on FPS. Useful in conjunction with --ai_training."));
+	print_help_option(_("--new_game_from_template=FILENAME"),
+	                  _("Directly create a new singleplayer game configured in the given file. An "
+	                    "example can be found in `data/templates/new_game_template`."));
+	print_help_option(
+	   _("--scenario=FILENAME"), _("Directly starts the map FILENAME as scenario map."));
+	print_help_option(_("--loadgame=FILENAME"), _("Directly loads the savegame FILENAME."));
+	print_help_option(_("--replay=FILENAME"), _("Directly loads the replay FILENAME."));
+	print_help_option(
+	   _("--script=FILENAME"), _("Run the given Lua script after initialization. Only valid with "
+	                             "--scenario, --loadgame, or --editor."));
+	print_help_option(
+	   /** TRANSLATORS: You may translate true/false, also as on/off or yes/no, but it HAS TO BE
+	      CONSISTENT with the translation in the widelands textdomain */
+	   _("--auto_roadbuild_mode=[true|false]"),
+	   _("Whether to enter roadbuilding mode automatically after placing a flag that is not "
+	     "connected to a road."));
+
+	std::cout << std::endl << _("Graphic options:") << std::endl << std::endl;
+	print_help_option(
+	   /** TRANSLATORS: You may translate true/false, also as on/off or yes/no, but it HAS TO BE
+	      CONSISTENT with the translation in the widelands textdomain */
+	   _("--fullscreen=[true|false]"), _("Whether to use the whole display for the game screen."));
+	print_help_option(_("--xres=[...]"), _("Width of the window in pixels."));
+	print_help_option(_("--yres=[...]"), _("Height of the window in pixels."));
+	print_help_option(_("--maxfps=[5 ...]"), _("Maximal optical framerate of the game."));
+
+	std::cout << std::endl
+	          << _("Options for the internal window manager:") << std::endl
+	          << std::endl;
+	print_help_option(
+	   /** TRANSLATORS: You may translate true/false, also as on/off or yes/no, but it HAS TO BE
+	      CONSISTENT with the translation in the widelands textdomain */
+	   _("--animate_map_panning=[true|false]"), _("Should automatic map movements be animated."));
+	print_help_option(_("--border_snap_distance=[0 ...]"),
+	                  _("Move a window to the edge of the screen when the edge of the window comes "
+	                    "within this distance from the edge of the screen."));
+	print_help_option(
```

Comment:
Same here, maybe better `=n` and refer to 'n' in the text, e.g. "Snap window to the edge of the screen if it is n pixels away of it", or so :-D

Same for all options which needs a number value.